core/domain/suggestion_registry.py

```diff
@@ -680,14 +682,46 @@ def __init__(
         self.target_version_at_submission = target_version_at_submission
         self.author_id = author_id
         self.change = question_domain.QuestionSuggestionChange(change)
-        # Update question_state_data_schema_version here instead of surfacing
-        # the version in the frontend.
-        self.change.question_dict['question_state_data_schema_version'] = (
-            feconf.CURRENT_STATE_SCHEMA_VERSION)
         self.score_category = score_category
         self.language_code = language_code
         self.last_updated = last_updated
         self.image_context = feconf.IMAGE_CONTEXT_QUESTION_SUGGESTIONS
+        self._updated_change()
+
+    def _updated_change(self):
```

Comment:
This should be a present-tense verb. Perhaps update_change_to_latest_state_schema_version() or similar?

Reply:
Done!